docs(inputs.prometheus): Add example for prometheus timestamps (#13362)
influxdata/telegraf: plugins/inputs/prometheus/README.md

prometheus,cpu=cpu2,url=http://example.org:9273/metrics cpu_usage_user=2.119071644805144 1505776751000000000
prometheus,cpu=cpu3,url=http://example.org:9273/metrics cpu_usage_user=1.5228426395944945 1505776751000000000
```

### Output with timestamp included

> Assuming a prometheus metric example including a timestamp

Reviewer: ```suggestion
Below is an example of a Prometheus metric which includes a timestamp:
```

```text
# TYPE test_counter counter
test_counter{label="test"} 1 1685443805885
```

> we get the following metric

Reviewer: ```suggestion
Telegraf will generate the following metric:
```

```text
test_counter,address=127.0.0.1,label=test counter=1 1685443805885000000
```

using the standard configuration

```toml

> [inputs.prometheus]]

Reviewer: ```suggestion
[[inputs.prometheus]]
```

  ## An array of urls to scrape metrics from.
  urls = ["http://localhost:2019/metrics"]
```

> **Please note:** The prometheus timestamp is in **millisecond** precision!

Reviewer: This is the case, even if the default precision in telegraf is 1 second?

Author: The whole point for this PR is that prometheus sends its timestamp in milliseconds, i.e. the timestamp in the incoming message is in milliseconds. Help in a better wording is appreciated! ;-)

Reviewer: Right I know - I guess we need to state that the timestamp is always in milliseconds, and it ignores the precision set by the telegraf agent. How about:

**Please note:** Metrics generated by the prometheus plugin are always generated with **millisecond** precision! Even if the agent sets precision to another value.

Thoughts?

Author: > [...] and it ignores the precision set by the telegraf agent.

That is not completely true. The incoming timestamp is in milliseconds but the precision is honored it's just that it doesn't get better than milliseconds. So if you set precision to seconds the resulting metrics will strip the milliseconds...

Reviewer: Your original proposal reads like it will always produce metrics with millisecond precision. My concern is that as a user you will use this plugin, see that the results are in seconds and file an issue because the default agent level setting is seconds, meaning by default results are in seconds. 

The only way a user sees the millisecond precision is if their agent or input precision setting is in milliseconds or smaller:

How about:

> Please note: Metrics generated by the prometheus plugin are generated with millisecond precision at most, but the default agent level precision setting reduces this to seconds. Setting the precision setting at the agent or plugin level to milliseconds or smaller will report millisecond precision.
